configs/opencv-ovms/demos/instance_segmentation/python/instance_segmentation_demo.py

```diff
@@ -116,32 +121,43 @@ def get_model(model_adapter, configuration):
     return MaskRCNNModel(model_adapter, configuration)
 
 
-def print_raw_results(boxes, classes, scores, frame_id):
-    log.debug('  -------------------------- Frame # {} --------------------------  '.format(frame_id))
-    log.debug('  Class ID | Confidence |     XMIN |     YMIN |     XMAX |     YMAX ')
+def print_raw_results(boxes, classes, labels, scores, frame_id):
+    # log.debug('  -------------------------- Frame # {} --------------------------  '.format(frame_id))
+    # log.debug('  Class ID | Confidence |     XMIN |     YMIN |     XMAX |     YMAX ')
+    data = []
     for box, cls, score in zip(boxes, classes, scores):
-        log.debug('{:>10} | {:>10f} | {:>8.2f} | {:>8.2f} | {:>8.2f} | {:>8.2f} '.format(cls, score, *box))
-
-def ResizeWithAspectRatio(image, width=None, height=None, inter=cv2.INTER_AREA):
-    dim = None
-    (h, w) = image.shape[:2]
-
-    if width is None and height is None:
-        return image
-    if width is None:
-        r = height / float(h)
-        dim = (int(w * r), height)
-    else:
-        r = width / float(w)
-        dim = (width, int(h * r))
-
-    return cv2.resize(image, dim, interpolation=inter)
+        # log.debug('{:>10} | {:>10f} | {:>8.2f} | {:>8.2f} | {:>8.2f} | {:>8.2f} '.format(cls, score, *box))
+        # print(box)
+        det_label = labels[cls] if labels and len(labels) >= cls else '#{}'.format(cls)
+        json_object = {
+            "label": det_label,
+            "score": float(score),
+            "xmin": float(box[0]),
+            "ymin": float(box[1]),
+            "xmax": float(box[2]),
+            "ymax": float(box[3])
+        }
+        data.append(json_object)
+    json_array = json.dumps(data)
+    return json_array  
 
 def main():
       
     args = build_argparser().parse_args()
 
     cap = open_images_capture(args.input, args.loop)
+    
+    containerName = os.environ.get("CONTAINER_NAME", "object_detection") 
+    client = None
+    # Connect to MQTT
+    if args.mqtt:
+        try:
+            client = mqtt.Client(containerName)
+            client.connect(args.mqtt,1883,60)    
+            client.loop_start()
+            print("connected to mqtt")
+        except: 
+            print("cannot connect to mqtt")
```

Comment:
```suggestion
            print("cannot connect to mqtt on port 1883")
```
we might also want to consider to print out the error from except error: